pylib/System/TestDef.py

```diff
@@ -582,6 +582,20 @@ def configTest(self):
                 print("ERROR: %s"%l)
             sys.exit(1)
 
+        # find all the sections that match the wild card and expand them
+        # this is simple wild carding, ie *text, text*, *text* and * 
+        # should all work, text*text will not
+        expsec = []
+        cpsections = list(sections)
+        for sec in cpsections:
+            if '*' in sec:
+                modsec = sec.replace('*','')
+                for s in self.config.sections():
+                    if modsec in s:
+                        expsec.append(s)
+                sections.remove(sec)
+        sections = sections + expsec
```

Comment:
not sure if it matters, but this does reorder what is in ```sections```

Reply:
it does, it adds the expanded wildcard'd sections to the end of the sections list.

but for --skipsections it will not matter.  

Not sure if that will matter for --sections, but i think it won't matter either because we add based on the order of sections as generated from the ini file